### ISSUE TITLE
Pos1 key jumps to very first char in row

### ISSUE THREAD

**Issue by BernhardNinaus** [dbeaver/dbeaver#9393]
**Is your feature request related to a problem? Please describe.**
When your cursor is at the last char at a given row, before new line, and you press "Pos1" then you jump to the very first char (including tabulator and sapce).

**Describe the solution you'd like**
The cursor should jump to the first char, which is not a tabulator or space.
E.g.:
```sql
    --"_" is cursor position
    --From positon   
    ,tab.PERSONID_

    --To position
    _,tab.PERSONID

   --When pressing again then:
_   ,tab.PERSONID
```

**Comment by uslss** [comment]
thanks for suggestion

**Comment by uslss** [comment]
will be fixed in scope of #9414
